### ISSUE TITLE
Error: [$rootScope:infdig] 10 $digest() iterations reached.

### ISSUE THREAD

**Issue by robertjd** [stormpath/stormpath-sdk-angularjs#20]
While building your Angular application with this module, you may encounter this error:

```
Error: [$rootScope:infdig] 10 $digest() iterations reached. Aborting!
Watchers fired in the last 5 iterations: []
```

This happens when your default view (`main` if you are following our Angular Guide) is using the `waitForUser` option.  The cause is this upstream bug: https://github.com/angular-ui/ui-router/issues/600 

**Workaround**:

Instead of this:

```
$urlRouterProvider
  .otherwise('/');
```

Use this:

```
$urlRouterProvider.otherwise( function($injector, $location) {
    var $state = $injector.get("$state");
    $state.go("main"); //redirect to a 404 page
});
```

If you are using our Angular Guide to create a project from scratch, that code will go into `client/app.js`

**Comment by ashishbajaj99** [comment]
Great, thanks a lot Robert!

**Comment by agupta1989** [comment]
Yes, it was eating my brain. Thanks a lot @robertjd . Would you please also explain why this was happening with simple $urlRouterProvider.other('/') call and not with the alternative ?

**Comment by robertjd** [comment]
I'd look at the linked issue above, for UI-Router - it explains the nuances of the issue.
